assets/app/scripts/controllers/create/nextSteps.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+/* jshint unused: false */
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:CreateFinishedController
+ * @description
+ * # CreateFinishedController
+ * Controller of the openshiftConsole
+ */
+angular.module("openshiftConsole")
+  .controller("CreateFinishedController", function($scope, $http, $routeParams, DataService, $q, $location, TaskList, $parse, Navigate, $filter, imageObjectRefFilter, failureObjectNameFilter) {
+    var displayNameFilter = $filter('displayName');
+    var watches = [];
+
+    $scope.emptyMessage = "Loading...";
+    $scope.alerts = [];
+    $scope.baseUrl = location.origin;
+    $scope.projectName = $routeParams.project;
+    $scope.unfilteredBuildConfigs = {};
```

Comment:
if you aren't supporting filtering on the page (which I can't imagine you would on this page), just call it $scope.buildConfigs